channels/ke.m3u

```diff
@@ -1,17 +1,49 @@
-https://stream-06.ix7.dailymotion.com/sec(uK1M-ag2Z_MzR7ZEmmKyDXI_wCqDQusqgardAY5XCr0)/dm/3/x6lvncs/d/live-3.m3u8
-#EXTINF:-1 tvg-id="Kameme TV Kenya" tvg-name="Kameme TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/ke-kameme-tv-1383.jpg" group-title="",Kameme TV Kenya
-https://stream-05.dc3.dailymotion.com/sec(nitAgzxq_qCa0CId9XzFH2v-wTlR6kxRn5k_zLyiuaA)/dm/3/x6ol8sj/d/live-3.m3u8
-#EXTINF:-1 tvg-id="KBC Kenya" tvg-name="KBC" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/KBC-1.jpg" group-title="",KBC Kenya
-https://stream-01.dc3.dailymotion.com/sec(86G48EQSWKUUFPhHXEV5xmRLVrEbPYAHOZrsRqUifEc)/dm/3/x74211t/s/live-2.m3u8
-#EXTINF:-1 tvg-id="KTN Kenya" tvg-name="KTN Kenya" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/KTN-1.jpg" group-title="",KTN Kenya
+#EXTINF:-1 tvg-id="EbruTV.ke" tvg-name="Ebru TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/download-1.jpg" tvg-url="" group-title="General",Ebru TV
+https://tvsn-i.akamaihd.net/hls/live/261837/tvsn/tvsn_750.m3u8
+#EXTINF:-1 tvg-id="EbruTV.ke" tvg-name="Ebru TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/download-1.jpg" group-title="General",Ebru TV
+https://iptv--iptv.repl.co/streamlink?url=https://dai.ly/x67n3k1
+#EXTINF:-1 tvg-id="EbruTV.ke" tvg-name="Ebru TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/download-1.jpg" group-title="General",Ebru TV
+http://salvahe.co/iptv-query?url=https://dai.ly/x67n3k1
+#EXTINF:-1 tvg-id="InooroTV.ke" tvg-name="Inooro TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/InooroTV-1.jpg" group-title="General",Inooro TV
+https://iptv--iptv.repl.co/streamlink?url=https://dai.ly/x7ttbvq
+#EXTINF:-1 tvg-id="InooroTV.ke" tvg-name="Inooro TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/InooroTV-1.jpg" group-title="General",Inooro TV
+http://salvahe.co/iptv-query?url=https://dai.ly/x7ttbvq
+#EXTINF:-1 tvg-id="K24.ke" tvg-name="K24" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/K24-1.jpg" tvg-url="" group-title="General",K24 [Not 24/7]
+https://5f4db0f94b000.streamlock.net/k24/myStream/chunklist_w333466763.m3u8
+#EXTINF:-1 tvg-id="K24.ke" tvg-name="K24" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/K24-1.jpg" group-title="General",K24
+https://iptv--iptv.repl.co/streamlink?url=https://dai.ly/x6lvncs
+#EXTINF:-1 tvg-id="K24.ke" tvg-name="K24" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/K24-1.jpg" group-title="General",K24
+http://salvahe.co/iptv-query?url=https://dai.ly/x6lvncs
```

Comment:
The link doesn't work.

Reply:
![image](https://user-images.githubusercontent.com/5430116/119546190-51cefe80-bd9c-11eb-98e8-f07ef061ebb1.png)
Do I have to try the others?

Reply:
The Salvahe links are dead.